nss-c31-unusual-prestige/nutshell
# Av chat display current messages

# Description

Created function to pull messages from the API and post them to the DOM. Requires HtmlBUilder.js and apiManager.js, as well as a the database from json.messages.example to be present.

Applies to issue #5 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

-git fetch --all
-git checkout av-chat-display-current-messages
-find database.json.messages.example and remove the .messages.example (or copy the contents into an existing database.json).
-in main.js add line "messageManager.printMessages("messages");
-remember to save
-run grunt
-navigate to localhost in browser.
-see that messages have been appended to DOM.
-check code in case there are overlooked errors and confirm readability.


# Checklist:
